src/parse.js

```diff
@@ -1,9 +1,9 @@
+import fs from 'node:fs';
```

Comment:
ℹ️ In case the synchronous module is not used, the async module can be imported directly
```javascript
import fs from 'node:fs/promises';
```

Reply:
I purposly define imports as above in case I need to use both async and async APIs (uncommon but happens sometimes).